### PR TITLE
u9fs: 20110513 -> unstable-2020-11-21

### PR DESCRIPTION
**Pull request by siraben**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by siraben** [comment]
Result of `nixpkgs-review pr 104576` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>u9fs</li>
  </ul>
</details>

File: pkgs/servers/u9fs/default.nix [hunk 1]
```diff
@@ -1,24 +1,26 @@
...
 stdenv.mkDerivation {
-  name = "u9fs-20110513";
+  pname = "u9fs";
+  version = "unstable-2020-11-21";
+
   src = fetchhg {
-    url = "https://bitbucket.org/plan9-from-bell-labs/u9fs";
-    rev = "9474edb23b11";
+    url = "https://code.9front.org/hg/plan9front";
+    rev = "6eef4d6a9bce";
```
Comment (SuperSandro2000) [inline pkgs/servers/u9fs/default.nix:9]: Can you use the full hash? Making this hash ambiguous is quite easy.

File: pkgs/servers/u9fs/default.nix [hunk 2]
```diff
@@ -1,24 +1,26 @@
...
-      homepage = "http://plan9.bell-labs.com/magic/man2html/4/u9fs";
-      license = licenses.free;
-      maintainers = [ maintainers.ehmry ];
-      platforms = platforms.unix;
-    };
+  meta = with stdenv.lib; {
+    description = "Serve 9P from Unix";
+    homepage = "http://plan9.bell-labs.com/magic/man2html/4/u9fs";
+    license = licenses.free;
+    maintainers = [ maintainers.ehmry ];
```
Comment (SuperSandro2000) [inline pkgs/servers/u9fs/default.nix:23]:
```suggestion
    maintainers = with maintainers; [ ehmry ];
```

**Comment by SuperSandro2000** [comment]
@ehmry because ofborg is having problems.

File: pkgs/servers/u9fs/default.nix (hunk 1 above)
Comment (siraben) [reply pkgs/servers/u9fs/default.nix:9]: This is the full hash, hg doesn't have a longer one.

File: pkgs/servers/u9fs/default.nix (hunk 2 above)
Comment (siraben) [reply pkgs/servers/u9fs/default.nix:23]: Is this preferred?

File: pkgs/servers/u9fs/default.nix (hunk 1 above)
Comment (SuperSandro2000) [reply pkgs/servers/u9fs/default.nix:9]: Just use git ™️

File: pkgs/servers/u9fs/default.nix (hunk 2 above)
Comment (SuperSandro2000) [reply pkgs/servers/u9fs/default.nix:23]: it makes changing maintainers easier. I prefer it.

**Pull request merged**